src/main/cpp/mechanisms/noteManager/decoratormods/noteManager_backupManualLaunch_State.cpp

```diff
@@ -34,24 +33,32 @@ using namespace std;
 
 /// @class ExampleForwardState
 /// @brief information about the control (open loop, closed loop position, closed loop velocity, etc.) for a mechanism state
-noteManagerbackupManualLaunchState::noteManagerbackupManualLaunchState ( std::string stateName,
-        int stateId,
-        noteManagerbackupManualLaunchStateGen *generatedState,
-        noteManager *mech ) : State ( stateName, stateId ), m_genState ( generatedState ), m_mechanism ( mech )
+noteManagerbackupManualLaunchState::noteManagerbackupManualLaunchState(std::string stateName,
+																	   int stateId,
+																	   noteManagerbackupManualLaunchStateGen *generatedState,
+																	   noteManager *mech) : State(stateName, stateId), m_genState(generatedState), m_mechanism(mech)
 {
 }
 
 void noteManagerbackupManualLaunchState::Init()
 {
-	Logger::GetLogger()->LogData ( LOGGER_LEVEL::PRINT, string ( "ArrivedAt" ), string ( "noteManagerbackupManualLaunchState" ), string ( "init" ) );
-
+	Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, string("ArrivedAt"), string("noteManagerbackupManualLaunchState"), string("init"));
 	m_genState->Init();
+
+	m_mechanism->Elevator->SetControlConstants(0, *m_mechanism->percentOutput);
```

Comment:
need to move percentOutput to be a private variable and then access it via an accessor